src/dipdup/config/abi_etherscan.py

```diff
@@ -0,0 +1,24 @@
+from typing import Literal
+
+from pydantic.dataclasses import dataclass
+
+from dipdup.config import AbiDatasourceConfig
+from dipdup.config import HttpConfig
+
+API_URL = 'https://api.etherscan.io/api'
```

Comment:
I'd not hardcode that

Reply:
`EtherscanDatasourceConfig` has `url` field to overwrite this value. Renamed URL consts to represent it better.